### PR TITLE
add link

### PR DESCRIPTION
**Pull request by rkelson**
for FEMA Flood Map Service Center

### CODE REVIEW

File: data/water/flood-areas/index.html [hunk 1]
```diff
@@ -69,6 +69,7 @@ <h5>Related Resources</h5>
     <ul class="dotless">
       <li><a href="https://snmapmod.snco.us/fmm/document/fema-flood-zone-definitions.pdf">Definitions of FEMA Flood Zone
           Designations</a>
+      <li><a href="https://msc.fema.gov/portal/home">FEMA Flood Map Service Center</a>
```
Comment (stdavis) [inline data/water/flood-areas/index.html:72]:
```suggestion
      <li><a href="https://snmapmod.snco.us/fmm/document/fema-flood-zone-definitions.pdf">Definitions of FEMA Flood Zone
          Designations</a></li>
      <li><a href="https://msc.fema.gov/portal/home">FEMA Flood Map Service Center</a></li>
```

**Review by stdavis — CHANGES_REQUESTED** [review]
List item tags (`li`) should have a closing tag. In HTML all tags should have a corresponding closing tag unless they have a `/` in the opening tag like this: `<hr />`.

**Comment by steveoh** [comment]
> List item tags (`li`) should have a closing tag. In HTML all tags should have a corresponding closing tag unless they have a `/` in the opening tag like this: `<hr />`.

This isn't exactly true. I'll edit with the html5 spec which shows otherwise

**Comment by stdavis** [comment]
> This isn't exactly true. li closing tags are optional. [w3.org/TR/2014/REC-html5-20141028/syntax.html#optional-tags](https://www.w3.org/TR/2014/REC-html5-20141028/syntax.html#optional-tags) as well as many other elements.

Oh yes, I forgot that we've been through this before. I see it as a best practice but it's not a big deal to me.

**Review by steveoh — APPROVED** [review]

**Pull request merged**